dbfit-java/core/src/main/java/dbfit/fixture/ExecuteProcedureExpectException.java

```diff
@@ -0,0 +1,39 @@
+package dbfit.fixture;
+
+import dbfit.api.DBEnvironment;
+import dbfit.api.DbEnvironmentFactory;
+import dbfit.util.ExpectedBehaviour;
+
+public class ExecuteProcedureExpectException extends ExecuteProcedure {
+    private boolean excNumberDefined = false;
+    private int excNumberExpected;
+
+    public ExecuteProcedureExpectException() {
+        this.environment = DbEnvironmentFactory.getDefaultEnvironment();
+    }
+
+    public ExecuteProcedureExpectException(DBEnvironment dbEnvironment, String procName,
+                            int expectedErrorCode) {
+        this.procName = procName;
+        this.environment = dbEnvironment;
+        this.excNumberDefined = true;
+        this.excNumberExpected = expectedErrorCode;
+    }
+
+    public ExecuteProcedureExpectException(DBEnvironment dbEnvironment, String procName) {
+        this.procName = procName;
+        this.environment = dbEnvironment;
+        this.excNumberDefined = false;
+    }
+
+    @Override
+    protected ExpectedBehaviour getExpectedBehaviour() {
+        if (!excNumberDefined) return ExpectedBehaviour.ANY_EXCEPTION;
+        return ExpectedBehaviour.SPECIFIC_EXCEPTION;
```

Comment:
May we simplify that a bit? It should be possible to fit it in a single 1-line operator.